core/src/main/java/org/apache/brooklyn/core/mgmt/rebind/RebindIteration.java

```diff
@@ -988,7 +988,7 @@ protected void setCatalogItemId(BrooklynObject item, String catalogItemId) {
             } catch (Exception e) {
                 Exceptions.propagateIfFatal(e);
             }
-            return new ClassLoaderUtils(reflections.getClassLoader()).loadClass(jType);
+            return new ClassLoaderUtils(reflections.getClassLoader(), managementContext).loadClass(jType);
```

Comment:
I fail to see how this fixes the test (looking at the `ClassLoaderUtils` code). If `jType` has a prefix it should be able to load it anyway. If not then it will fail.

Looks good otherwise.